src/game/logic/map/polygontrigger.cpp

```diff
@@ -21,28 +24,417 @@
 PolygonTrigger *PolygonTrigger::s_thePolygonTriggerListPtr;
 #endif
 
-bool PolygonTrigger::Parse_Polygon_Triggers_Data_Chunk(DataChunkInput &file, DataChunkInfo *info, void *userdata)
+int PolygonTrigger::s_currentID;
+
+PolygonTrigger::PolygonTrigger(int initial_allocation) :
+    m_nextPolygonTrigger(nullptr),
+    m_points(nullptr),
+    m_numPoints(0),
+    m_sizePoints(0),
+    m_riverStart(0),
+    m_exportWithScripts(false),
+    m_isWaterArea(false),
+    m_isRiver(false),
+    m_isShownInLayer(true),
+    m_isSelected(false)

```

Comment:
Some members such as `m_radius` and `m_boundsNeedsUpdate` are not initialized here.

Reply:
fixed.